cmake/modules/Findnuma.cmake

```diff
@@ -0,0 +1,16 @@
+# - Find libnuma
```

Comment:
i'd suggest rename `Findnuma.cmake` to `FindNUMA.cmake`. as like ALSA, BLAS, NUMA is also an abbreviation. or, at least, we should be consistent on the naming of exposed variables. like `Findfoo.cmake` => `foo_INCLUDE_DIR` .